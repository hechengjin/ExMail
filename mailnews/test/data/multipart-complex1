Content-Type: multipart/mixed; boundary="boundary"

This shouldn't appear.
--boundary
Content-Type: application/octet-stream
Content-Transfer-Encoding: base64

VGhpcyBpc24ndCByZWFsbHkgYW4gYXBwbGljYXRpb24vb2N0ZXQtc3RyZWFtLiA7KQ=='

--boundary
Content-Type: image/png
Content-Transfer-Encoding: base64

TmVpdGhlciBpcyB0aGlzIGFuIGltYWdlL3BuZy4=

--boundary
Content-Type: multipart/related; boundary="boundary2"

--boundary2
Content-Type: text/html

<html><head>This part should be returned.</head></html>

--boundary2--

--boundary
Content-Type: text/plain

This part shouldn't.

--boundary

Neither should this part!

--boundary--
