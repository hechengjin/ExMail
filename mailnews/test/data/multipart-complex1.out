This part should be returned.